Makefile

```diff
@@ -46,3 +49,24 @@ test-bindings: $(PKG_BINDINGS_DIR)
 
 clean-bindings:
 	rm -rf $(PKG_BINDINGS_DIR)
+
+image: Dockerfile
+	DOCKER_BUILDKIT=1 \
+	$(DOCKER) build \
+	    --progress=plain \
+	    --tag $(BUILDIMAGE) \
+	    .
+
+docker-%: image
+	@echo "Running 'make $(*)' in docker container $(BUILDIMAGE)"
+	$(DOCKER) run \
+		--rm \
+		-e PWD \
+		-v $(PWD):$(PWD) \
+		-w $(PWD) \
+	    $(BUILDIMAGE) \
+			make $(*)
+
+# Override the target for removing the bindings directory recursively from the container
+docker-clean-bindings: clean-bindings
+	@echo "Skipping containerized removal"
```

Comment:
Why do we want to override this and not do it from docker?

Reply:
Since we're mapping in `$(pwd)` I didn't want to inadvertently delete `/` due to an unset variable.